### PR TITLE
HYC-1554 - Get work page action buttons showing again

### PR DESCRIPTION
**Pull request by bbpennel**
https://jira.lib.unc.edu/browse/HYC-1554

* Updates related view files to match the changes from hyrax 3
* Updates workflow presenter
* Adds cleaning to one of the tests that was failing due to the work action buttons not showing, so that the test full passes.
* Fix rubocop issues in migration files

### CODE REVIEW

File: app/views/hyrax/base/show.html.erb [hunk 1]
```diff
@@ -22,28 +20,54 @@
...
           <div class="col-sm-3 text-center">
+            <%# [hyc-override] Remove restriction on showing representative media info when iiif viewer present %>
             <%= render 'representative_media', presenter: @presenter, viewer: false %>
+            <%# [hyc-override] Changes end here %>
             <%= render 'citations', presenter: @presenter %>
             <%= render 'social_media' %>
           </div>
           <div class="col-sm-9">
+            <%# [hyc-override] Remove description and display it interleaved in the metadata %>
+            <%# <%= render 'work_description', presenter: @presenter %>
```
Comment (sharonluong) [inline app/views/hyrax/base/show.html.erb:38]: what is this commented out line here for?

Comment (bbpennel) [reply app/views/hyrax/base/show.html.erb:38]: This line exists in the hyrax version of the file
https://github.com/samvera/hyrax/blob/v3.4.2/app/views/hyrax/base/show.html.erb#L27
I figured it'd make it a bit easier to compare the files in the future to figure out what we changed if I left in the deleted line commented out. But I could could remove it and just leave the hyc-override tag as the reminder

Comment (sharonluong) [reply app/views/hyrax/base/show.html.erb:38]: oh! i guess it's not obvious at first glance that the commented out portion is the hyrax code. maybe a little note would suffice?

Comment (bbpennel) [reply app/views/hyrax/base/show.html.erb:38]: I updated the override message to indicate that the commenting out is related to the override, does that help?

**Review by sharonluong — APPROVED** [review]

**Pull request merged**